drivers/scheduler/scheduler.go

```diff
@@ -178,6 +147,9 @@ type Driver interface {
 	// ResizeVolume resizes all the volumes of a given context
 	ResizeVolume(*Context, string) ([]*volume.Volume, error)
 
+	// CalculateVolumesSize calculates volumes size based on autopilot rule
+	CalculateVolumesSizes(*Context) (map[string]uint64, error)
```

Comment:
CalculateVolumesSizes doesn't need to a an interface. It's just an internal helper method that is being called from InspectVolumes.